includes/definitions/allworx.yaml

```diff
@@ -0,0 +1,10 @@
+os: allworx
```

Comment:
allworx is the vendor name, do they not have an OS name these products run or a group name for the product line?

Reply:
The have both the X (legacy) series and the Connect (current) series of telephone switches but they don't have an OS name and the software is unified between the models. I am happy to change it but I don't know what we'd change it to.

Any suggestions?

Reply:
Since Allworx makes a line of Layer 3 smart switches I forgot about, I'll rename this OS to allworx_voip